devel/icu/Portfile

```diff
@@ -40,8 +40,8 @@ checksums                   rmd160  4537a0ce1879b24cdee150862615a7a899d6fa05 \
 if {${os.major} >= 11 || ${subport} eq "${name}-bootstrap" || ${subport} eq "${name}58"} {
     configure.python        /usr/bin/python
 } else {
-    depends_build-append    port:python27
-    configure.python        ${prefix}/bin/python2.7
+    depends_build-append    port:python27-bootstrap
+    configure.python        ${prefix}/libexec/libcxx-bootstrap/bin/python2.7
```

Comment:
I don't think this is necessary. With the change to the portgroup, python should always build with an Xcode compiler or clang-3.4.

Reply:
On Mac OS X 10.6,
* icu depends on python27
* python27 depends on libedit
* libedit depends on xattr
* xattr depends on unzip
* unzip depends on clang-3.7
* clang-3.7 depends on libxml2
* libxml2 depends on icu
* ...

Reply:
Ah, I forgot to add clang_dependency to xattr when adding its use to libedit.

Reply:
xattr fixed in dd1bbe7.

Reply:
I am not sure if https://github.com/macports/macports-ports/commit/dd1bbe7e0a42a26411a11f79a42e9d6b7634f248 was meant to be a first step or not, but after making the same change, I still get the same circular dependency as before.
xattr still depends on unzip

Reply:
unzip fixed in a49aa48.

Reply:
I removed the change to the Python dependency in the pull request.
Thank you for finding a better solution.